staging/src/k8s.io/mount-utils/resizefs_linux_test.go

```diff
@@ -0,0 +1,112 @@
+			expectError: false,
+			fsType:      "ext4",
+		},
+		{
+			name:        "block size not present - ext4",
+			devicePath:  "/dev/test1",
+			blocksize:   0,
+			blockCount:  0,
+			cmdOutput:   "Filesystem volume name:   cloudimg-rootfs\nLast mounted on:          /\nFilesystem UUID:          testUUID\nFilesystem magic number:  0xEF53\nFilesystem revision #:    1 (dynamic)\nFilesystem features:      has_journal ext_attr resize_inode dir_index filetype needs_recovery extent 64bit\nDefault mount options:    user_xattr acl\nFilesystem state:         clean\nErrors behavior:          Continue\nFilesystem OS type:       Linux\nInode count:              3840000\nReserved block count:     0\nFree blocks:              5514413\nFree inodes:              3677492\nFirst block:              0\nFragment size:            4096\nGroup descriptor size:    64\nReserved GDT blocks:      252\nBlocks per group:         32768\nFragments per group:      32768\nInodes per group:         16000\nInode blocks per group:   1000\nFlex block group size:    16\nMount count:              2\nMaximum mount count:      -1\nCheck interval:           0 (<none>)\nLifetime writes:          180 GB\nReserved blocks uid:      0 (user root)\nReserved blocks gid:      0 (group root)\nFirst inode:              11\nInode size:\t          256\nRequired extra isize:     32\nDesired extra isize:      32\nJournal inode:            8\nDefault directory hash:   half_md4\nDirectory Hash Seed:      Test Hashing\nJournal backup:           inode blocks\nChecksum type:            crc32c\nChecksum:                 0x57705f62\nJournal features:         journal_incompat_revoke journal_64bit journal_checksum_v3\nJournal size:             64M\nJournal length:           16384\nJournal sequence:         0x00037109\nJournal start:            1\nJournal checksum type:    crc32c\nJournal checksum:         0xb7df3c6e",
```

Comment:
doesn't this read better with `` format of writing go strings?

Reply:
Good point, updated